amethyst_network/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "amethyst_network"
+version = "0.0.1"
+authors = ["Joël Lupien (Jojolepro) <jojolepromain@gmail.com>"]
+description = "Amethyst network crate"
+keywords = ["network", "specs", "ecs", "amethyst", "serialization"]
+categories = ["game-engines"]
+
+documentation = "https://www.amethyst.rs/doc/develop/doc/amethyst_network/index.html"
+homepage = "https://www.amethyst.rs/"
+repository = "https://github.com/amethyst/amethyst"
+license = "MIT/Apache-2.0"
+
+[dependencies]
+amethyst_assets = { path = "../amethyst_assets", version = "0.2" }
+amethyst_core = { path = "../amethyst_core", version = "0.1" }
+cgmath = { version = "0.14", features = ["eders"] }
```

Comment:
Pssst, we re-export cgmath in amethyst_core.

If you need version 0.14 for a specific reason feel free to disregard this comment.

Reply:
fixed